modules/tinymce/src/plugins/table/test/ts/browser/ui/TableDialogTest.ts

```diff
@@ -573,10 +599,10 @@ describe('browser.tinymce.plugins.table.TableDialogTest', () => {
 
     it('TINY-9837: Should apply cellpadding and bordercolor to cells if they have been modified',
       () => testApplyDataToCells({ cellpadding: '3px', bordercolor: '#FF0000' }, [
-        '<table style="border-collapse: collapse; width: 500px; height: 500px; border-width: 2px; border-color: #FF0000; border-style: dotted;" border="1"><colgroup><col style="width: 50%;"><col style="width: 50%;"></colgroup>',
+        '<table style="width: 500px; height: 500px; border: 2px dotted rgb(255, 0, 0); border-collapse: collapse;" border="1"><colgroup><col style="width: 50%;"><col style="width: 50%;"></colgroup>',
```

Comment:
Why was this changed from hex to RGB?

Reply:
Because [the new change](https://github.com/tinymce/tinymce/pull/8813/files#diff-278a6644e0ffb99813d0019eca11c186c502815d373ae299b603d6a10f208471L99) is to update the dom style instead of updating the style attribute. As a side effect of [dom.style.setProperty()](https://github.com/tinymce/tinymce/blob/2d8b95517a43d1f1967a2f3898371865a2fa18e4/modules/sugar/src/main/ts/ephox/sugar/api/properties/Css.ts#L21), hex is converted to rgb